packages/components/src/components/ConformityCredential/ConformityCredential.tsx

```diff
@@ -0,0 +1,210 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import _ from 'lodash';
+
+import { LoadingButton } from '@mui/lab';
+import { Table, TableBody, TableCell, TableRow, TableContainer, TableHead, Paper } from '@mui/material';
+import { uploadJson, generateUUID, getJsonDataFromConformityAPI } from '@mock-app/services';
+
+import { checkStoredCredentials, getCredentialByPath } from './utils.js';
+import { Status, ToastMessage, toastMessage } from '../ToastMessage/ToastMessage.js';
+import {
+  IConformityCredentialProps,
+  ICredentialRequestConfig,
+  IConformityCredential,
+} from '../../types/conformityCredential.types.js';
+
+export const STORAGE_KEY = 'conformityCredentials';
+
+export const ConformityCredential: React.FC<IConformityCredentialProps> = ({
+  credentialRequestConfigs,
+  storedCredentials,
+}) => {
+  const [conformityCredentials, setConformityCredentials] = useState<string>();
+  const [loading, setLoading] = useState<boolean>(false);
+  const [loadingButtonIndex, setLoadingButtonIndex] = useState<number>(-1);
```

Comment:
These states can optimize like 
```
const [loadingButtonIndexes, setLoadingButtonIndexes] = useState<number[]>([]);
```
When any button is loading, it will be set to the state

Reply:
Thanks for your recommendation, after trying to apply it, I think I should rename `loadingButtonIndex`. It's a state for disabling the button when loading and it is not related to loading.